liberty-archetype-webapp/src/test/resources/projects/it/archetype.properties

```diff
@@ -0,0 +1,5 @@
+groupId=test.wlp
+artifactId=ServletSample
+version=0.1-SNAPSHOT
+wlpRuntimeVersion=16.0.0.4
+wlpPluginVersion=1.3-SNAPSHOT
```

Comment:
This one is a test properties file and it doesn't have to be in sync with the one in archetype-metadata.xml.   It overrides the default value in archetype-metadata.xml.